frontend-vue/src/pages/analytics/Analytics.vue

```diff
@@ -295,16 +294,6 @@ function convertFilter(): ActiveFilter[] {
 
 const activeFilters = ref<ActiveFilter[]>(convertFilter());
 
-watch(
-  () => route.query,
-  () => {
-    const routeFilters = convertFilter();
-    if (!arrayContentEqual(activeFilters.value, routeFilters)) {
-      activeFilters.value = routeFilters;
```

Comment:
This check always returns false when route is updated since items of `activeFilters.value` are proxied/wrapped while items of `routeFilters` are just plain objects.